modules/osd-monitoring-configurable-monitoring-components.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-configuring-the-monitoring-stack.adoc
+
+[id="configurable-monitoring-components_{context}"]
+= Configurable monitoring components
+
+This table shows the monitoring components you can configure and the keys used to specify the components in the `user-workload-monitoring-config` `ConfigMap` objects:
+
+.Configurable monitoring components
+[options="header"]
+|===
+|Component |`user-workload-monitoring-config` `config map` key
```

Comment:
```
`user-workload-monitoring-config` `config map` key
```
only `key` is bold
should be
`user-workload-monitoring-config config map key`
then all the words are bold

Reply:
It would be better use 'user-workload-monitoring-config ConfigMap key'

Reply:
OK, changed it.